### PR TITLE
Integrated

### PR DESCRIPTION
**Pull request by mjanuszkiewicz-tt**

### CODE REVIEW

File: common/src/universal/run-tests.sh [hunk 1]
```diff
@@ -0,0 +1,22 @@
...
+docker run -d \
+  -e "discovery.type=single-node" \
+  -e "traveltime.app.id=id" \
+  -e "traveltime.api.key=key" \
+  -e "xpack.security.enabled=false" \
+  --rm \
+  --name $IMAGE_NAME \
+  $IMAGE_NAME
+
+docker exec $IMAGE_NAME ./wait-for-startup.sh
```
Comment (ajakubauskas) [inline common/src/universal/run-tests.sh:18]: the path for these shell scripts seems to be incorrect

Comment (mjanuszkiewicz-tt) [reply common/src/universal/run-tests.sh:18]: These paths are paths inside the container relative to its working directory. When building the image I call `COPY path/to/script ./`, where `./` is the working directory of the container.

Comment (ajakubauskas) [reply common/src/universal/run-tests.sh:18]: nvm, the files seem to be there but docker still fails to run them

Comment (ajakubauskas) [reply common/src/universal/run-tests.sh:18]:
```
+ docker exec v0.2-snapshot_5.5-test ./wait-for-startup.sh
OCI runtime exec failed: exec failed: container_linux.go:380: starting container process caused: exec: "./wait-for-startup.sh": permission denied: unknown
```

**Pull request merged**